apps/ima/translations/src/en/fields.json

```diff
@@ -17,5 +17,34 @@
         "hint": "You cannot give immigration advice. You must answer all the questions as if you are the person named on the duty to remove letter."
       }
     }
+  },
+  "temporary-permission-reasons": {
+    "legend": "What are your reasons for applying for temporary permission to stay in the UK?",
+    "hint": "Select all that apply.",
+    "options": {
+      "human-rights": {
+        "label": "human rights"
+      },
+      "other-international-obligations": {
+        "label": "other international obligations"
+      },
+      "exceptional-circumstances": {
+        "label": "exceptional circumstances"
+      }
+    }
+  },
+  "temporary-permission": {
+    "legend": " ",
```

Comment:
Removed the legend property and updated the classname to be visually hidden.

Reply:
On second thought, for screen reader purposes, it would be better to have it has  `"legend": "Are there any reasons you should be granted temporary permission to stay in the UK?`" but kept visually hidden otherwise a screenreader could read it as 'fields.temporary-permission.legend' instead of the actual question.

Reply:
Added the legend text and kept it as visually hidden.